### PR TITLE
documentation(EJ2-845801): committed new features in core docs.

### PR DESCRIPTION
**Pull request by indumathi-murali**
committed new features in core docs.

### CODE REVIEW

**Comment by indumathi-murali** [comment]
coresampletested

**Comment by indumathi-murali** [comment]
MVC sample tested - 
[MVCSamplesTested.zip](https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/files/12712037/MVCSamplesTested.zip)

**Comment by indumathi-murali** [comment]
Core sample link - httpswww.syncfusion.comdownloadssupportdirecttracgeneralzeMapsCoreSample2071173444

**Comment by indumathi-murali** [comment]
MVC Sample link - https://www.syncfusion.com/downloads/support/directtrac/general/ze/WebApplication4-944811054

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 1]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-array-table/tagHelper %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-array-table.cs" %}
+{% include code-snippet/heatmap/appearance/template-array-table/template-array-table.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET Core HeatMap chart with data label template with array table binding](./images/appearance/data-label-template-with-array-table-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:373]: ASP.NET Core HeatMap chart with data label template and array table binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 2]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight razor tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-array-table/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-array-table.cs" %}
+{% include code-snippet/heatmap/appearance/template-array-table/template-array-table.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET MVC HeatMap chart with data label template with array table binding](./images/appearance/data-label-template-with-array-table-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:386]: ASP.NET MVC HeatMap chart with data label template and array table binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 3]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-array-cell/tagHelper %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-array-cell.cs" %}
+{% include code-snippet/heatmap/appearance/template-array-cell/template-array-cell.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET Core HeatMap chart with data label template with array cell binding](./images/appearance/data-label-template-with-array-cell-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:406]: ASP.NET Core HeatMap chart with data label template and array cell binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 4]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-json-table/tagHelper %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-json-table.cs" %}
+{% include code-snippet/heatmap/appearance/template-json-table/template-json-table.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET Core HeatMap chart with data label template with JSON table binding](./images/appearance/data-label-template-with-JSON-table-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:443]: ASP.NET Core HeatMap chart with data label template and JSON table binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 5]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight razor tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-json-table/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-json-table.cs" %}
+{% include code-snippet/heatmap/appearance/template-json-table/template-json-table.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET MVC HeatMap chart with data label template with JSON table binding](./images/appearance/data-label-template-with-JSON-table-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:456]: ASP.NET MVC HeatMap chart with data label template and JSON table binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 6]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-json-cell/tagHelper %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-json-cell.cs" %}
+{% include code-snippet/heatmap/appearance/template-json-cell/template-json-cell.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET Core HeatMap chart with data label template with JSON cell binding](./images/appearance/data-label-template-with-JSON-cell-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:476]: ASP.NET MVC HeatMap chart with data label template and JSON table binding

File: ej2-asp-core-mvc/heatmap-chart/appearance.md [hunk 7]
```diff
@@ -334,3 +334,158 @@ The format of the data label, such as currency, decimal, percent etc. can be cha
...
+{% tabs %}
+{% highlight razor tabtitle="CSHTML" %}
+{% include code-snippet/heatmap/appearance/template-json-cell/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Template-json-cell.cs" %}
+{% include code-snippet/heatmap/appearance/template-json-cell/template-json-cell.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![ASP.NET MVC HeatMap chart with data label template with JSON cell binding](./images/appearance/data-label-template-with-JSON-cell-binding.png)
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/heatmap-chart/appearance.md:489]: ASP.NET MVC HeatMap chart with data label template and JSON cell binding

**Review by sabari-senthamaraikannan — APPROVED** [review]

**Pull request merged**